internal/pkg/client/oras/oras.go

```diff
@@ -285,6 +287,9 @@ func ImageSHA(ctx context.Context, uri string, ociAuth *ocitypes.DockerAuthConfi
 
 	// ensure that we received an image manifest descriptor
 	if desc.MediaType != ocispec.MediaTypeImageManifest {
+		if desc.MediaType == manifest.DockerV2Schema2MediaType {
+			return "", errors.New("please use docker:// instead of oras:// to pull the image")
```

Comment:
This message should say something about an error before there's a suggestion of what to do.  Start it with "unexpected docker media type received; "

Reply:
thanks for the comment! reflected in this commit https://github.com/apptainer/apptainer/pull/703/commits/ed8c9af19331d20eb636b00bb05d4a7517f03ac8